src/NServiceBus.Persistence.AzureTable/TableClientExtensions.cs

```diff
@@ -0,0 +1,37 @@
+﻿namespace NServiceBus.Persistence.AzureTable
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Linq.Expressions;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Azure;
+    using Azure.Data.Tables;
+
+    static class TableClientExtensions
```

Comment:
These might not be needed in case we can use `ToListAsync` as pointed out before